frontend_gen.py

```diff
@@ -348,10 +447,90 @@
                                 width=component["width"],
                                 route=component["route"],
                                 link=link,
+                                channel=channel,
                             )
                         )
+                        import_set.add(
+                            "const TableView = React.lazy(() => import('../Table/TableView'))"
+                        )
+                    elif re.match(r"^slider.*$", component["type"]):
+                        p.write(
+                            slider_template.format(
+                                component_name=component_name,
+                                x=component["x"],
+                                y=component["y"],
+                                height=component["height"],
+                                width=component["width"],
+                                route=component["route"],
+                                channel=component["channel"],
+                                channelList=(
+                                    f"channelList={{{component['''channels''']}}}"
+                                    if "channels" in component
+                                    else ""
+                                ),
+                                store=(
+                                    "store={Object.assign({}, this.state.store)}"
+                                    if "channels" in component
+                                    else ""
```

Comment:
Perhaps we can probably push this a bit higher too?

Reply:
Same as the other comment. The component only gets access to the store if channels are specified. This also needs to be decided on a component basis.

Reply:
Sounds good. :+1: